src/common/constants.ts

```diff
@@ -70,28 +70,14 @@ export const DEFAULT_MAINNET_SERVER =
   process.env.NEXT_PUBLIC_MAINNET_API_SERVER ||
   'https://stacks-node-api.stacks.co';
 
+export const LOCALHOST_NETWORK_SERVER = 'http://localhost:3999';
```

Comment:
Should this be `DEFAULT_DEVNET_SERVER` to better associate it to what we call devnet everywhere?